package.json

```diff
@@ -66,6 +66,7 @@
     "oref0_init_pump_comms.py": "./bin/oref0_init_pump_comms.py",
     "oref0-log-shortcuts": "./bin/oref0-log-shortcuts.sh",
     "oref0-mdt-trend": "./bin/oref0-mdt-trend.js",
+    "oref0-mdt-update": "./bin/oref0-mdt-update.js",
```

Comment:
It's oref0-mdt-update.sh, not .js.  And you'll want to change the references in the code from /root/src/oref0/bin/oref0-mdt-update.sh to just oref0-mdt-update as well.